src/components/Toolbar.tsx

```diff
@@ -192,13 +192,18 @@ const Toolbar = ({ dark, fontSize, toolbarOverlay, scrollPrioritized, showTopCon
             {shortcutIds.map(id => {
               const { name, svg, exec, isActive, canExecute } = shortcutById(id)!
               const isButtonActive = isActive?.(store.getState) || !canExecute || canExecute(store.getState)
+
               // TODO: type svg correctly
               const SVG = svg as React.FC<Icon>
               return (
                 <div
                   key={name}
                   id={id}
-                  style={pressingToolbarId === id ? { paddingTop: '10px' } : {}}
+                  style={{
+                    paddingTop: pressingToolbarId === id ? '10px' : '',
+                    cursor: isButtonActive ? 'pointer' : 'default',
+                    pointerEvents: isButtonActive ? 'auto' : 'none',
```

Comment:
This will incorrectly disable the button when the shortcut is executable but inactive, such as when the Table View is not active on a thought, but can be activated. 

Instead, we want to disable the button only if it cannot be executed.

Reply:
But if the button is executable, shouldn't it be active too?

Reply:
No, there are some commands that show active if a meta attribute is on the thought. For example:

```js
- a
  - =note
    - hello
- b
```

If the cursor is on `a`, then the Note button will be *executable* and *active*. If the cursor is on `b`, the Note button will be *executable* and *inactive*.

It is confusing because it is ambiguous. I am overloading the `isButtonActive` state to handle two different things. I tried to explain it here, but it is understandably confusing: https://github.com/cybersemics/em/pull/1304#discussion_r659219440

It's like `isButtonActive` reflects if `isActive` returns true for the given shortcut, or if `isActive` is not defined at all, with the added exception that if the shortcut is not executable, it should not show as active.

Reply:
I re-read your comment and tried to follow it in app. It makes sense. I will make the change. Thanks